dashboard/public/pixelation/pixelation.js

```diff
@@ -476,3 +535,5 @@ function startOverConfirmed() {
   drawGraph();
   formatBitDisplay();
 }
+
+pixelationInit();
```

Comment:
You have suggested this, for _dialog.html.haml. That code is (the closest thing we have to) a library and is pulled into lots of places, and I took your suggestion to wrap that code. I agree with Josh that it is less worth it in this case because the globals exposed in this file will only exist for pixelation, so there's relatively low risk of other code accessing any globals, especially now that there's no JS left in pixelation.html.